src/mmrelay/matrix/prefixes.py

```
@@ -0,0 +1,599 @@
+
+    Parameters:
+        format_string (str): The format string to validate (uses str.format syntax).
+        available_vars (dict): Mapping of placeholder names to sample values used to test formatting.
+
+    Returns:
+        tuple: (is_valid, error_message). is_valid is True if formatting succeeds, False otherwise. error_message is the exception message when invalid, or None when valid.
+    """
+    try:
+        # Test format with dummy data
+        format_string.format(**available_vars)
+        return True, None
```

Ruff (0.15.9)

[warning] Consider moving this statement to an `else` block

(TRY300)